sound/usb/quirks.c

```diff
@@ -2028,6 +2028,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case USB_ID(0x249c, 0x9326): /* M2Tech Young MkIII */
 	case USB_ID(0x2616, 0x0106): /* PS Audio NuWave DAC */
 	case USB_ID(0x2622, 0x0041): /* Audiolab M-DAC+ */
+	case USB_ID(0x2622, 0x0061): /* LEAK Stereo 230 */
```

Comment:
Please submit these [patches upstream](https://www.kernel.org/doc/html/latest/process/submitting-patches.html) so everyone benefits :)

If you need help submitting patches feel free to contact me (jelle@archlinux.org), I can also submit it on behalf of you.